_episodes/01-intro.md

```diff
@@ -133,19 +170,18 @@ parameter: the name of the file we want to read data from. This
 parameter needs to be a character string or
 [string](reference.html#string), so we put it in quotes.
 
-Our call to `csvread` read our file, and printed the data inside
-to the screen. And adding a semicolon rendered it even less useful---
-we have no way to modify those values
-or compute with them. To do that, we need to assign the array to a
+Our call to `csvread` read our file, and printed the data 
+to the screen so we can view it, however, we have no way to modify those values
+or compute with them. Adding a semicolon rendered it even less useful. To work with our data we need to assign the set of values to a
 [variable](reference.html#variable).
 
 ~~~
 patient_data = csvread('inflammation-01.csv');
 ~~~
 {: .matlab}
 
-A variable is just a name for a piece of data or *value*.
-Variable names must begin with a letter, and can contain
+A variable is like a labeled box containing  a piece of data or *value*.
```

Comment:
The "labeled box" analogy is nice for functions, but a variable really is a name for a piece of data or *value*. I'm in favour of removing the *just*.